src/components/PeoplePicker/PeoplePicker.tsx

```diff
@@ -507,7 +507,7 @@ export class PeoplePicker extends React.Component<IPeoplePickerProps, IPeoplePic
                   <button className='ms-PeoplePicker-resultAction' onClick={ () => {
                       this._removeSelectedPersona( selectedPersonas.indexOf(child) );
                   }}>
-                    <i className='ms-Icon ms-Icon--Cancel'></i>
+                    <i className='ms-Icon ms-Icon--Clear'></i>
```

Comment:
Generally yes, we use `Clear` for this particular character. I'm having a tough time telling the difference between these two though. @leddie24, why the switch?